app/src/js/content.js

```diff
@@ -23,13 +23,13 @@ window.addEventListener("load", () => {
         mediumScripts.length > 0 ||
         SCRIPT_SRC_REGEX.test(window.location.hostname)
     ) {
-        console.log("Is a medium blog !");
+        console.log("Unlimited Medium extension: medium blog detected");
```

Comment:
Let's remove the log all together. This is used mainly for debugging. In the future, I might add a proper logging mechanism and this line would be replaced by a call to `logger.debug` which would print only in dev mode and not in production.